ipatests/test_integration/test_upgrade.py

```diff
@@ -98,6 +98,24 @@ def clear_sysupgrade(host, *sections):
     host.put_file_contents(statefile, out.getvalue())
 
 
+def get_main_krb_rec_dn(domain):
+    return DN(
+        ('idnsname', '_kerberos'),
+        ('idnsname', domain.name + '.'),
+        ('cn', 'dns'),
```

Comment:
Probably better form to use ipalib.constants.container_dns rather than ('cn', 'dns').